### PR TITLE
Adding 4 templates, their tests, and modified index file

### PR DESCRIPTION
**Pull request by rishabh5j**
##### ISSUE TYPE
 - New Template Pull Request

##### COMPONENT
arista_eos_show_environment_cooling
arista_eos_show_environment_temperature
arista_eos_show_interfaces_transceiver_detail
cisco_nxos_show_environment_temperature

##### SUMMARY
Added new templates, their tests and modified index

### CODE REVIEW

File: tests/cisco_nxos/show_environment_temperature/cisco_nxos_show_environment_temperature.raw [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+Temperature
+-------------------------------------------------------------------------
+Module  Sensor             MajorThresh   MinorThres   CurTemp     Status
+                            (Celsius)     (Celsius)   (Celsius)         
+-------------------------------------------------------------------------
+1       ASIC                101           95           52          ok          
+1       Front-Middle(D1)    62            56           38          ok          
+1       Front-Left  (D2)    52            44           33          ok          
+1       Back        (D3)    48            42           28          ok """
```
Comment (itdependsnetworks) [inline tests/cisco_nxos/show_environment_temperature/cisco_nxos_show_environment_temperature.raw]: Minor, but can you remove triple quote?

**Comment by itdependsnetworks** [comment]
Thanks a lot for your PR!

2 things:
 - Can you set values to be upper case?
 - Can you look at equivalents for transceiver in nxos, xr and ensure that the names of the values match up (they may be already)

File: templates/arista_eos_show_interfaces_transceiver_detail.template [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  ^mA:\smilliamperes,\s+
+  ^\+\+\s:\shigh\salarm,\s+
+  ^A2D\sreadouts\s+
+  ^The\sthreshold\svalues\sare\scalibrated\.
+  ^\s+High\sAlarm\s+\S+
+  ^\s+${type}\s+Threshold\s+ -> Values
+  ^\s+${type}\s+Threshold\s+ -> Values
+  ^\s+${type}\s+Threshold\s+ -> Values
+  ^\s+${type}\s+Threshold\s+ -> Values
+  ^\s+${type}\s+Threshold\s+ -> Values
```
[the remark this answers is not in the document]
Comment (rishabh5j) [reply templates/arista_eos_show_interfaces_transceiver_detail.template]: You are not missing anything, My bad. This template was restructured which caused this. Correcting in next iteration.

**Comment by itdependsnetworks** [comment]
Apologies, one more request. Does it make sense to remove the percentage character from being captured?

File: tests/cisco_nxos/show_environment_temperature/cisco_nxos_show_environment_temperature.raw (hunk 1 above)
Comment (rishabh5j) [reply tests/cisco_nxos/show_environment_temperature/cisco_nxos_show_environment_temperature.raw]: Done.

**Comment by itdependsnetworks** [comment]
What happened to this PR? I suddenly see a whole other branch you are including.

**Comment by rishabh5j** [comment]
@itdependsnetworks  Travis ci is failing checks on this PR, but not for any of my templates. Can you pls check and merge?

**Comment by itdependsnetworks** [comment]
What happened here? Looks like a merge issue or something? You are taking in someone else’s commits

**Comment by rishabh5j** [comment]
Abandoning and will raise another PR.

**Pull request closed without merging**